lib/spack/spack/test/relocate.py

```diff
@@ -241,7 +241,7 @@ def test_file_is_relocatable(source_file, is_relocatable):
 
 @pytest.mark.requires_executables('patchelf', 'strings', 'file')
 def test_patchelf_is_relocatable():
-    patchelf = spack.relocate._patchelf()
+    patchelf = os.path.realpath(spack.relocate._patchelf())
```

Comment:
@gartung Do you remember which was the purpose of this test?

Reply:
Apparently added in #13430 by @marcmengel with an explanation of its use in the description